tests/Oryx.BuildImage.Tests/BenvTest.cs

```diff
@@ -93,7 +91,8 @@ public BenvTest(ITestOutputHelper output)
                 .ToString();
 
             // Act
-            var result = _dockerCli.Run(SlimBuildImageName, "/bin/bash", "-c", script);
+            var image = _imageHelper.GetTestImage("build-slim", "latest");
```

Comment:
Isn't `build-slim` not being renamed to `build:slim` as part of this PR?

Reply:
This will be handled in a separate PR once these changes are merged.